pos-justpos.dvc

```diff
@@ -0,0 +1,17 @@
+cmd: python spacy_convert/converter.py --output-path data/spacy_convert/pos-justpos.json
+  --conversion-map-filepath data/tagmap_data/nkjp2us.json
```

Comment:
Czy teraz nkjp2us to jest mapa konwersji samych posów?

Reply:
Tak